frame/pablo/src/lib.rs

```diff
@@ -104,7 +104,7 @@ pub mod pallet {
 	use sp_std::{collections::btree_map::BTreeMap, vec::Vec};
 
 	#[derive(RuntimeDebug, Encode, Decode, MaxEncodedLen, Clone, PartialEq, Eq, TypeInfo)]
-	pub enum PoolInitConfiguration<AccountId, AssetId, BlockNumber> {
+	pub enum PoolInitConfiguration<AccountId, AssetId, BlockNumber, StakingPoolId> {
```

Comment:
This forces us to always track staking pools for pool configurations which may or may not be required(LBPs don't need it). We can possibly introduce a new `create_pool` extrinisic to create pablo pools stake pools as they only need to be created for those pools that we want to incentivize.